### PR TITLE
docs(release-notes): april release notes

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -19,58 +19,51 @@
 
 ## What's New?
 
-:material-star:** Database update required **
-An upgrade to the `rpsdb` database is required.
+:material-new-box:** New Feature: 802.1X Wireless Configuration Support **
 
-[See Upgrade Toolkit Version docs for SQL scripts to add the new table and columns](../Deployment/upgradeVersion/).
+For devices deployed in a secure enterprise network, configuring AMT to authenticate with the network is important so that connection can be maintained with the device even when the OS is not available. With this release, we now support configuring AMT with certificate-based 802.1x configurations (EAP-TLS, PEAPv1/EAP-GTC, EAP-FAST/GTC, EAP-FAST/TLS) for wireless networks.  [Find more information about 802.1x configuration here](../References/EA/ieee8021xconfig/).
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:24]: Bad link, /Reference/ not /References/.  Also maybe we jump them right to Wireless:
```
../Reference/EA/ieee8021xconfig/#wireless-8021x-configuration
```

Also need to update the supported configs (only EAP-TLS and PEAPv0/EAP-MSCHAPv2)

File: docs/release-notes.md [hunk 2]
```diff
@@ -19,58 +19,51 @@
...
 
-:material-new-box:** New Feature: 802.1X Wired Configuration Support **
+:material-hammer:** Quality of Life Improvement: Added -f (force) flag for maintenance commands **
 
-For devices deployed in a secure enterprise network, configuring AMT to authenticate with the network is important so that connection can be maintained with the device even when the OS is not available.  With this release, we now support configuring AMT with certificate based 802.1x configurations (EAP-TLS, PEAPv1/EAP-GTC, EAP-FAST/GTC, EAP-FAST/TLS).  [Find more information about 802.1x configuration here](../References/EA/ieee8021xconfig/). 
+We have added the -f (force) flag for maintenance commands to allow users to bypass the password check RPS performs. This will help customers who need to run a maintenance task on AMT but have not yet added the device credentials to the secret store.
```
Comment (bwendlandt-intel) [inline docs/release-notes.md:28]: We can link them to here:
```
../Reference/RPC/commandsRPC/#maintenance
```

**Review by bwendlandt-intel — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (matt-primrose) [reply docs/release-notes.md:24]: I didn't think we support MSCHAPv2 yet.  that will be later

Comment (bwendlandt-intel) [reply docs/release-notes.md]: Yeah was just checking myself.  I'll open PR to update 8021x docs then to remove.

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**